Sources/CreateXCFramework/Zipper.swift

```diff
@@ -106,6 +109,28 @@ struct Zipper {
     func clean (file: Foundation.URL) throws {
         try FileManager.default.removeItem(at: file)
     }
+
+    #if swift(>=5.7)
+    private func checksum(forBinaryArtifactAt path: AbsolutePath) throws -> String {
```

Comment:
`checksum(forBinaryArtifactAt:)` is moved from `Workspace` to `Workspace.BinaryArtifactsManager`.
Unfortunately, `workspace.binaryArtifactsManager` is concealed from externals.

So I ported the same implementation from the original SwiftPM codes.

https://github.com/apple/swift-package-manager/blob/release/5.7/Sources/Workspace/Workspace%2BBinaryArtifacts.swift#L390-L413